webview/src/experiments/components/table/styles.module.scss

```diff
@@ -80,17 +80,9 @@ $badge-size: 0.85rem;
   background-color: $header-border-color;
```

Comment:
Looked at storybook, and apparently, storybook applies table element selectors to the plots stories. With the code outside of the `.experiments` wrapper, the plots stories are now breaking. 

Are we already aware of this issue in storybook? I'm not sure why it's happening, but if it's hard to fix, we could:

* move everything back into an `experiments` wrapper. While annoying to have another wrapper around everything, it would be the quickest way to fix the problem. 
* Don't select elements at all, just rely on classes. We'd need to add some extra classes like `.td` and `.th` and update the file accordingly.

Reply:
Discuss this on our call. Need to understand why these were being applied (seems like a bug in the way we've setup the stories). What is shown in the extension should be what is shown in Storybook.

Reply:
I agree with Matt, this looks like a bug in how we setup the stories. Couldn't find where we directly import the table styles, but we directly import the plots one. We should probably clean that up and find out where these styles are coming from. That said, I think relying on classes instead of elements is good practices. I'm the one to blame here for that. During the react-table upgrade I changed all the `div`s to table elements and removed the `.td`, `.th` classes because they weren't on the right elements. I should have moved them instead.